flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -701,17 +702,22 @@ public RocksDBNativeMetricOptions getMemoryWatcherOptions() {
 	}
 
 	/**
-	 * Gets the thread number will used for downloading files from DFS when restore.
+	 * Gets the number of threads used to transfer files while snapshotting/restoring.
 	 */
-	public int getNumberOfRestoringThreads() {
-		return numberOfRestoringThreads == UNDEFINED_NUMBER_OF_RESTORING_THREADS ?
-			RocksDBOptions.CHECKPOINT_RESTORE_THREAD_NUM.defaultValue() : numberOfRestoringThreads;
+	public int getNumberOfTransferingThreads() {
+		return numberOfTransferingThreads == UNDEFINED_NUMBER_OF_TRANSFERING_THREADS ?
+			CHECKPOINT_TRANSFER_THREAD_NUM.defaultValue() : numberOfTransferingThreads;
 	}
 
-	public void setNumberOfRestoringThreads(int numberOfRestoringThreads) {
-		Preconditions.checkArgument(numberOfRestoringThreads > 0,
-			"The number of threads used to download files from DFS in RocksDBStateBackend should > 0.");
-		this.numberOfRestoringThreads = numberOfRestoringThreads;
+	/**
+	 * Sets the number of threads used to transfer files while snapshotting/restoring.
+	 *
+	 * @param numberOfTransferingThreads The number of threads used to download files from DFS while restoring.
```

Comment:
`used to download files from DFS while restoring` -> `to transfer files while snapshotting/restoring`